doc/sphinx/changes.rst

```diff
@@ -14,6 +14,91 @@ Version 8.12
 Summary of changes
 ~~~~~~~~~~~~~~~~~~
 
+|Coq| version 8.12 integrates many quality-of-life improvements,
+in particular with respect to notations, scopes and implicit arguments,
+along with many bug-fixes and a reworked reference manual. The main changes
```

Comment:
```suggestion
along with many bug fixes and improvements to the reference manual. The main changes
```

"reworked" suggests that we're done but there's still a lot more ahead

Reply:
"improvements" is too weak to express these changes though: each version contains its lot of improvements to the refman. If you don't like "reworked" maybe "restructured" works better.

Reply:
To me, "restructured" only suggests the reorganization of sections with the new outline but not the detail changes for the syntax.  (restructure: "to change the makeup, organization, or pattern of")

How about "major improvements"?  "extensive improvements"?  "monumental improvements" :-) ?

Reply:
"major improvements" sounds good

Reply:
Yep, major sounds more appropriate.